README.md

```diff
@@ -84,6 +84,17 @@ Delete cookie:
 Cookies.remove('name');
 ```
 
+Please notice that removing unexisting cookie does not raise any exception nor return any value
+
+```javascript
+var whatWasTheValue = Cookies.remove('nothing');
+if (whatWasTheValue === undefined) {
+  console.log("I will be excuted");
+} else {
+  console.log("I will never be executed");
```

Comment:
It's really nice you appreciate my voice here, thanks :)

I like your approach to keep guides as short as possible but IMHO your new comments may confuse people even more than `fail`. 

The problem is that current `fail` comment is used for the first time in context of `"valid path of current page"`  so even adding super descriptive comment there is not intuitive enough to communicate that even removing Cookie with proper "path" argument also "fails"

I would just add block that every .remove call ends up with undefined, successful or not

```
Cookies.remove('name'); // returns undefined
Cookies.remove('nothing'); // also returns undefined
```